### ISSUE TITLE
508-defect-2 [COGNITION]: Resources and support search button should not be disabled

### ISSUE THREAD

**Issue by joshkimux** [department-of-veterans-affairs/va.gov-team#26413]
# [508-defect-2](https://github.com/department-of-veterans-affairs/va.gov-team/blob/master/platform/accessibility/guidance/defect-severity-rubric.md#508-defect-2)

## Feedback framework

- **❗️ Must** for if the feedback must be applied 
- **⚠️ Should** if the feedback is best practice 
- **✔️ Consider** for suggestions/enhancements 

## Definition of done

1. Review and acknowledge feedback.
1. Fix and/or document decisions made.
1. Accessibility specialist will close ticket after reviewing documented decisions / validating fix.

## Point of Contact

**VFS Point of Contact:** Josh

## User Story or Problem Statement

As a Veteran with cognitive disabilities, I expect to be able to receive clear feedback when submitting the form.

## Details

1. When buttons are inactive, some users-- particularly those with cognitive disabilites- may not know how to activate them. For example, if someone were to type in a short acronym like "VA" to search and the button doesn't become accessible, they may think the search field is broken.

<img width="841" alt="Screen Shot 2021-06-22 at 10 28 49 AM" src="https://user-images.githubusercontent.com/14154792/122942997-a38b8a00-d344-11eb-96a5-bb0cb7c33fb7.png">

2. Inactive buttons _are still_ accessible to folks using a mobile screen reader, which means they can still swipe to the button using screen reader focus and hear "Search" despite it being inactive. If they double tap it and receive no feedback or insufficient results, they may become confused.

3. The other two search fields have search buttons that are always active. Resources and support should follow this pattern to be consistent.

## Acceptance Criteria

- [x] Search button is always active
- [x] If a certain parameter must be met before conducting a search, it is communicated to the user via error message so they can know how to fix it

## WCAG or Vendor Guidance

![image](https://user-images.githubusercontent.com/14154792/122943777-4d6b1680-d345-11eb-84a1-bfea8ad24785.png)

**Comment by zacharymorel** [comment]
Did you mean to assign @kelsonic ?

**Comment by kelsonic** [comment]
I'm assuming so! 🙂 fysa @brianalloyd

**Comment by brianalloyd** [comment]
Thanks @kelsonic for the loop in on this issue.

**Comment by denisecoveyduc** [comment]
Great! Sounds like you guys have already caught this one. This is part of the staging review feedback I mentioned over slack @brianalloyd. You will see this on the list coming your way. 

@srsuddath @mgleo and I were discussing this one today and the redirect will be removed with #24992 - current plan is next sprint S51 when we update the user messaging #25264

**Comment by brianalloyd** [comment]
Thanks @denisecoveyduc its all coming together now.

**Comment by brianalloyd** [comment]
Reviewed and Validated. Closing ticket in support.
